Makefile.pre.in

```diff
@@ -893,15 +873,6 @@ regen-token:
 		$(srcdir)/Grammar/Tokens \
 		$(srcdir)/Lib/token.py
 
-.PHONY: regen-keyword
-regen-keyword:
```

Comment:
So how is keyword.py kept up to date?

Reply:
I think it'd be better to keep this target and support keyword generation in the pegen parser generator, so that it's able to create `keyword.py`. I could do that today. @pablogsal Do you agree and if so, could I just push to your branch?

Reply:
I was planning to do another PR in this issue with a new file in pegen that generates the keywords but I didn't want to add this to this PR because is already massive and is easier to review the less "new lines" in the diff it has

Reply:
Do you already have something or should I do it?

Reply:
> Do you already have something or should I do it?

I have started something already, but feel to create another PR if you want :)

Use the same issue as this PR is using, please

Reply:
Cross-referencing the PR here: https://github.com/python/cpython/pull/20800